README.md

```diff
@@ -41,6 +41,42 @@ Fedora Remix for WSL is provided on a community-support basis. There is no suppo
 
 Whitewater Foundry devs monitor the Fedora Remix for WSL GitHub issues page; however, users who require dedicated end-user support for WSL should consider [Pengwin](https://github.com/WhitewaterFoundry/Pengwin) instead. Businesses and other organizations that would like to receive ongoing professional support should [e-mail us](mailto:enterprise@whitewaterfoundry.com) or visit [our website](https://www.whitewaterfoundry.com/wlinux-enterprise-edition/) to learn more about Pengwin Enterprise.
 
+### In-place Upgrading
+You can upgrade your existing installation of Fedora Remix with the following steps. 
+
+First, backup your installation:
+
+`wsl --export fedoraremix fedoraremix_backup.tar.gz`
+
+Then proceed:
+
+> NOTE: DO NOT PERFORM THE UPGRADE STARTING FEDORA REMIX WITH SYSTEMD; IT WILL RUIN YOUR INSTALLATION
+
+#Be sure that systemd is disabled
+`[ "$(grep -c "^systemd.*=.*true$" /etc/wsl.conf)" -ne 0 ] && sudo sed -i "s/^systemd.*=.*true$/systemd=false/" /etc/wsl.conf && wsl.exe --terminate ${WSL_DISTRO_NAME}`
+
+```bash
+next_version=$(echo "${VERSION_ID}+1" | bc)
+
+update.sh 
+sudo dnf -y upgrade --refresh  
+sudo dnf -y install dnf-plugin-system-upgrade  
+sudo rpm --import https://src.fedoraproject.org/rpms/fedora-repos/raw/rawhide/f/RPM-GPG-KEY-fedora-${next_version}-primary  
```

Comment:
This step depends on how much a user ... tuned ... their nice and shiny and clean Fedora baseline installation.

A baseline Fedora 37 installation will not require this step; anyone who fancied things from rawhide (or other third-party repositories) should be held accountable for their own actions.

I would leave this out from generic instructions.